test/helpers/loki/receiver.go

```diff
@@ -61,6 +63,7 @@ func NewReceiver(ns, name string) *Receiver {
 			"-server.grpc-max-recv-msg-size-bytes", "20971520",
 			"-distributor.ingestion-rate-limit-mb", "200",
 			"-distributor.ingestion-burst-size-mb", "200",
+			"-validation.discover-log-levels=false",
```

Comment:
Why does this fail now and didn't before this submission?

Reply:
Updated version of loki is the cause.   Robert is aware and confirmed this is necessary.